packages/espressocash_app/lib/features/helius/data/helius_api_client.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:injectable/injectable.dart';
+
+import '../../../di.dart';
+import '../../rpc/rpc_client.dart';
+import 'dto.dart';
+
+@injectable
+class HeliusApiClient {
+  HeliusApiClient({
+    @rpcUrl required Uri clientRpcUrl,
+  }) : _rpcClient = JsonRpcClient(baseUrl: clientRpcUrl);
+
+  final JsonRpcClient _rpcClient;
+
+  Future<GetPriorityFeeEstimateResponse?> getPriorityFeeEstimate(
```

Comment:
Let's leave this on the backend side – let's have the backend proxying the call.